templates/docs/examples/patterns/slider/slider-input.html

```diff
@@ -1,16 +1,40 @@
-{% extends "_layouts/examples.html" %}
-{% block title %}Slider / Input{% endblock %}
-
-{% block standalone_css %}patterns_slider{% endblock %}
-
-{% block content %}
-<div class="p-slider__wrapper">
-  <input class="p-slider" type="range" min="0" max="100" value="50" step="1" id="slider3">
-  <input class="p-slider__input" type="text" maxlength="3" id="slider3-input">
+{% extends "_layouts/examples.html" %} {% block title %}Slider / Input{% endblock %} {% block standalone_css %}patterns_slider{% endblock %} {% block content %} <div class="p-slider__wrapper">
```

Comment:
Prettier messed up the template formatting.
Could you get the top part template tags back to original formatting (one per line)?